addons/core/addon/services/clock-tick.js

```diff
@@ -25,9 +25,15 @@ export default class ClockTickService extends Service {
   frequency = 1000;
 
   /**
-   * The timer interval ID
+   * Updates the value of `now`every `frequency` milliseconds.
    */
-  #timer;
+  _tick = dropTask(async () => {
```

Comment:
Would we ever see a scenario where this task is getting dropped?

Reply:
hmm ur right since these aren't tasks triggered by a user action